composer.json

```diff
@@ -16,15 +16,18 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
```

Comment:
Since this is for a major release anyway, 5.6 should be dropped, and current stable should be preferred